Automattic/wp-calypso
# People: fix prop type warning when searching for a team member

Partial fix for #20222. 

<img width="1279" alt="screen shot 2017-12-11 at 10 47 31" src="https://user-images.githubusercontent.com/17325/33827890-d068d16c-de61-11e7-919a-f5309271e975.png">

When a search is performed, it's possible for the `label` prop to be an array.

### To test

1. Visit http://calypso.localhost:3000/people/team/<yoursite.com>
2. Search for a team member (try your username)
3. Ensure that the above warning does not appear in your console
